### PR TITLE
release 3.2.0

### PR DESCRIPTION
**Pull request by markuman**
##### SUMMARY
Release community.aws 3.2.0

##### ISSUE TYPE

- Bugfix Pull Request
- Docs Pull Request
- Feature Pull Request
- New Module Pull Request

##### COMPONENT NAME
entire collection

### CODE REVIEW

**Comment by markuman** [comment]
recheck

**Comment by alinabuzachis** [comment]
recheck

File: CHANGELOG.rst [hunk 1]
```diff
@@ -5,6 +5,60 @@ community.aws Release Notes
...
+- elb_target_group - add support for alb target_type and update documentation (https://github.com/ansible-collections/community.aws/pull/966).
+- elb_target_group - add support for setting load_balancing_algorithm_type (https://github.com/ansible-collections/community.aws/pull/1016).
+- rds_instance - add `choices` for valid engine value (https://github.com/ansible-collections/community.aws/pull/1034).
+- rds_subnet_group - add ``check_mode`` (https://github.com/ansible-collections/community.aws/pull/562).
+- rds_subnet_group - add ``tags`` feature (https://github.com/ansible-collections/community.aws/pull/562).
+
+Bugfixes
+--------
+
+- Add backoff retry logic to elb_application_lb_info (https://github.com/ansible-collections/community.aws/pull/977)
```
Comment (tremble) [inline CHANGELOG.rst:44]: Minor formatting niggle: this should be 
`- elb_application_lb_info - ....`

File: CHANGELOG.rst [hunk 2]
```diff
@@ -5,6 +5,60 @@ community.aws Release Notes
...
+
+Major Changes
+-------------
+
+- s3_bucket_notifications - refactor module to support SNS / SQS targets as well as the existing support for Lambda functions (https://github.com/ansible-collections/community.aws/issues/140).
+
+Minor Changes
+-------------
+
+- Added suport for retries (AWSRetry.jittered_backoff) for cloudfront_distribution (https://github.com/ansible-collections/community.aws/issues/296)
```
Comment (tremble) [inline CHANGELOG.rst:26]: Similar niggle here, should be `- cloudfront_distribution - ...`

**Review by tremble — APPROVED** [review]
Minor niggle on changelogs, I'll leave it up to you if you think we should correct it.

**Review by alinabuzachis — APPROVED** [review]

File: CHANGELOG.rst [hunk 3]
```diff
@@ -23,7 +23,7 @@ Major Changes
 Minor Changes
 -------------
 
-- Added suport for retries (AWSRetry.jittered_backoff) for cloudfront_distribution (https://github.com/ansible-collections/community.aws/issues/296)
+- cloudfront_distribution - Added support for retries (AWSRetry.jittered_backoff) (https://github.com/ansible-collections/community.aws/issues/296)
```
Comment (tremble) [inline CHANGELOG.rst:26]: This'll need updating in changelog.yaml too

**Review by tremble — CHANGES_REQUESTED** [review]

**Review by tremble — APPROVED** [review]

**Pull request merged**